datasets/flwr_datasets/resplitter/merge_resplitter_test.py

```diff
@@ -25,7 +25,7 @@
 
 
 class TestResplitter(unittest.TestCase):
```

Comment:
Should we rename to `TestPreprocessor`?

Reply:
I think for that I should divide the file so there would be one for merge_resplitter and one for preprocessor in general. I'll do after we confirm the directory renaming.